### PR TITLE
 autotest C546: Create new request for Page type is fixed

### PR DESCRIPTION
**Pull request by penfriend**

### CODE REVIEW

File: cypress/support/fragments/inventory/markItemAsMissing.js [hunk 1]
```diff
@@ -178,10 +178,12 @@ export default {
   },
 
   openHoldingsAccordion(holdingId) {
+    cy.wait(10000);
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/markItemAsMissing.js:181]: why we need so long waiter?

File: cypress/support/fragments/users/userDefaultObjects/loanDetails.js [hunk 2]
```diff
@@ -65,15 +65,15 @@ export default {
...
+  checkKeyValue(label, valueString) {
+    cy.expect(KeyValue(label, { value: including(valueString) }).exists());
   },
   checkStatus(status) {
     this.checkKeyValue('Item status', status);
   },
   checkLostDate(date) {
     this.checkDateValid(date);
-    this.checkKeyValue('Lost', DateTools.getFormattedDateWithTime(date));
+ //   this.checkKeyValue('Lost', DateTools.getFormattedDate({ date }));
```
Comment (IhorBohdan) [inline cypress/support/fragments/users/userDefaultObjects/loanDetails.js:76]: Is need comment here?

File: cypress/support/utils/dateTools.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import moment from 'moment';
 import getRandomStringCode from './genereteTextCode';
+
```
Comment (IhorBohdan) [inline cypress/support/utils/dateTools.js:3]: Need here that?)

Comment (penfriend) [reply cypress/support/utils/dateTools.js:3]: there should be a new line after the import statements

File: cypress/support/fragments/inventory/markItemAsMissing.js (hunk 1 above)
Comment (penfriend) [reply cypress/support/fragments/inventory/markItemAsMissing.js:181]: removed waiter

File: cypress/support/fragments/users/userDefaultObjects/loanDetails.js (hunk 2 above)
Comment (penfriend) [reply cypress/support/fragments/users/userDefaultObjects/loanDetails.js:76]: uncommented

**Comment by penfriend** [comment]
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/670/allure

**Review by TetianaParanich — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**